config/saml_settings.yml

```diff
@@ -50,7 +50,8 @@ assertion_consumer_service_url: http://localhost:4567/consume
 issuer: urn:gov:gsa:SAML:2.0.profiles:sp:sso:localhost
 idp_sso_target_url: http://localhost:3000/api/saml/auth
 idp_slo_target_url: http://localhost:3000/api/saml/logout
-idp_cert_fingerprint: 8B:D5:C2:E8:9A:2B:CE:B7:4B:95:50:BA:16:79:05:27:17:D1:D3:67
+#idp_cert_fingerprint: 8B:D5:C2:E8:9A:2B:CE:B7:4B:95:50:BA:16:79:05:27:17:D1:D3:67
+idp_cert_fingerprint: 'BD:21:A9:57:14:3C:A8:4F:3F:C5:E6:99:7F:97:F6:E4:7E:E7:1D:1F'
```

Comment:
I think we identified that the test cert was copied instead of the development cert.

Reply:
Ack! yes!

Reply:
fixed!